README.md

```diff
@@ -1,6 +1,39 @@
-# gridt-client
+<p align="center">
+  <a href="https://www.gridt.org">
+    <img alt="Gridt" title="Gridt" src="https://app.gridt.org/assets/gridt_logo.png" width="300">
+  </a>
+</p>
+<h4 align="center">The network for connecting social movements.</h4>
+<p align="center">
+  <a href="https://app.gridt.org">
+    <img src="https://img.shields.io/website?url=https%3A%2F%2Fapp.gridt.org" alt="website">
+  </a>
+  <a href="https://github.com/GridtNetwork/gridt-client/actions?query=workflow%3ACI">
+    <img src="https://github.com/GridtNetwork/gridt-client/workflows/CI/badge.svg?branch=main" alt="Build Status">
+  </a>
+  <a href="https://github.com/GridtNetwork/gridt-client/actions?query=workflow%3ACI">
+    <img src="https://img.shields.io/github/last-commit/GridtNetwork/gridt-client" alt="Build Status">
+  </a>
+</p>
```

Comment:
Why are you going for inline HTML?

Reply:
Correction: SO SHINY!

Reply:
Centering isn't possible with plain Markdown. That's why.

Reply:
Sensible.